UI/src/view/point_asset/massTransitStopForm.js

```diff
@@ -806,9 +897,14 @@
         var allProperties = selectedMassTransitStopModel.getProperties();
         var properties;
 
-        if (isTerminalBusStop) {
+        /* don't change == to ===
+        * sometimes the value is int other times is string*/
+        if (busStopTypeSelected == 6 ) {
```

Comment:
Let's create a method to validate if it is terminal without spread the bus stops types codes in all code

Reply:
@marquesfm Check modification on this code